fredrondina96/Adopt-Dont-Shop-Part-2
# Caa user story 19

* Add user story 19 test
* Add application view page
* Add application view page route
* Add show method to applications_controller
* Pulled down master and had a bunch of merge conflicts. Resolved those by keeping everything.
* Once I pulled down master I had a bunch of errors and failing tests. I resolved those and everything is passing.
* Given these issues please review carefully to make sure nothing weird is going on.
